lib/csrf-hapi17.js

```diff
@@ -43,7 +43,7 @@ function csrfPlugin(server, options) {
   server.ext("onPreAuth", (request, h) => {
     const routeConfig = request.route.settings.plugins[pkg.name] || {};
 
-    csrf.process(
+    return csrf.process(
```

Comment:
Interesting. I don't think this was returning in the old hapi 16 plugin. Curious why we need it now?